### DISCUSSION TITLE
<LucideMapPin/> vs <MapPin/>

### DISCUSSION THREAD

**Discussion by manuelpoelzl** [lucide-icons/lucide#2414] [General]
Hey folks,

this question may seem kinda stupid, but is it recommended to use the components with the Lucide prefix, or the component without it?
I suppose there are two version in case Lucide is used alongside another icon library or am i wrong and there is something else to consider?

I'd really appreciate if someone could clarify this for me.

Thanks!

**Comment by karsa-mistmere** [comment]
It depends on your personal preference. Also see https://github.com/lucide-icons/lucide/pull/2328 where @ericfennis is working on making these imports better customisable.

I personally prefer prefixed or suffixed versions (depending on the stack), in order to avoid name collisions.

**Reply by manuelpoelzl** [reply]
Thanks for the clarification!
